packages/babel-helper-environment-visitor/src/index.ts

```diff
@@ -44,10 +44,13 @@ export default {
       }
     }
   },
-  "ClassProperty|ClassPrivateProperty"(
-    path: NodePath<t.ClassProperty | t.ClassPrivateProperty>,
-  ) {
+  Property(path) {
```

Comment:
The visitor now hooks on `Property` alias so that it supports older `@babel/types` without `ClassAccessorProperty` definitions.